source/blog/2017-08-14-ember-2-15-released.md

```diff
@@ -103,8 +103,31 @@ applications.
 
 ### Upgrading Ember CLI
 
-You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade
-your projects using `yarn` run:
+You may upgrade Ember CLI separately from Ember.js and Ember Data!
+There is a new experimental tool for Ember CLI upgrades called ember-cli-update.
+To use it, run this command to install it globally:
+
+```
+npm install -g ember-cli-update
+```
+
+Then run:
+
+```
+ember-cli-update
+```
+
+It runs your system git merge tool if it finds a conflict. This can be pretty
```

Comment:
I could also mention it runs the command `git mergetool`, unless that's too much info.